Tests/FunctionalTests/ClangModuleTests.swift

```diff
@@ -34,23 +34,23 @@ class ClangModulesTestCase: XCTestCase {
     func testSingleModuleFlatCLibrary() {
         fixture(name: "ClangModules/CLibraryFlat") { prefix in
```

Comment:
A bit late for this, but it would be nice if fixture() would return a structured object we could add helper methods onto, rather than just the prefix. This could allow us to clean up some of the boilerplate.